pkg/kubelet/kubelet_pods.go

```diff
@@ -1539,9 +1549,8 @@ func (kl *Kubelet) generateAPIPodStatus(pod *v1.Pod, podStatus *kubecontainer.Po
 		oldPodStatus = pod.Status
 	}
 	s := kl.convertStatusToAPIStatus(pod, podStatus, oldPodStatus)
-	if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
-		s.Resize = kl.determinePodResizeStatus(pod, s)
-	}
```

Comment:
It would suffice to add an else block to this if fgEnabled where it sets s.Resize = Infeasible in case it is non-empty (Proposed should be the only value when feature is enabled on API and resize has been requested)

Reply:
> in case it is non-empty

Could you clarify what should be non-empty here?

Reply:
FYI I went with my gut feeling in the update, so you can validate if it is correct.